docs/coverage-reporter/troubleshooting-coverage-cli-issues.md

```diff
@@ -0,0 +1,122 @@
+---
+description: Instructions or workarounds to overcome common issues while using Codacy Coverage Reporter CLI.
+---
+
+# Troubleshooting coverage CLI issues
+
+The sections below provide instructions or workarounds to overcome common issues while using Codacy Coverage Reporter CLI.
+
+## Commit SHA-1 hash detection {: id="commit-detection"}
+
+The Codacy Coverage Reporter automatically detects the SHA-1 hash of the current commit to associate with the coverage data when you're using one of the following CI/CD platforms:
+
+-   <span class="skip-vale">Appveyor</span>
+-   AWS CodeBuild
+-   Azure Pipelines
+-   <span class="skip-vale">Bitrise</span>
+-   <span class="skip-vale">Buildkite</span>
+-   Circle CI
+-   <span class="skip-vale">Codefresh</span>
+-   <span class="skip-vale">Codeship</span>
+-   Docker
+-   GitLab
+-   Greenhouse CI
+-   Heroku CI
+-   Jenkins
+-   Magnum CI
+-   Semaphore CI
+-   Shippable CI
+-   Solano CI
+-   TeamCity CI
+-   Travis CI
+-   <span class="skip-vale">Wercker CI</span>
```

Comment:
I'm thinking that instead of ignoring individual CI/CD platform names here, it could be best to just skip checking the entire section. :thinking: 

For instructing Vale to ignore multiple lines, there's a different syntax:

https://vale.sh/docs/topics/config/#markdown-amp-html